lib/opensusebasetest.pm

```diff
@@ -537,6 +537,8 @@ sub remount_tmp_if_ro {
 ## of wait_boot, e.g. see `git grep -l reboot | xargs grep -L wait_boot`
 sub post_fail_hook {
     my ($self) = @_;
+    return if is_serial_terminal();    # in case it is VIRTIO_CONSOLE=1 nothing below make sense
```

Comment:
after digging together with @Soulofdestiny we realized that this line will not do any harm, because in case someone select x11 console is_serial_terminal will return 0